### PR TITLE
Concourse

### LINKED ISSUE #4: Add Concourse Pipeline
Once we've created the lambda and the terraform, we should deploy these using a concourse pipeline

This issue can be closed when:
- [x] concourse runs tests and validates code
- [x] concourse deploys to demo
- [x] concourse deploys to prod
- [x] concourse has health checks
- [x] the lambdas are working and sending data to splunk in both prod and demo
- [x] the branch has been changed to master in the concourse pipeline

### PR DESCRIPTION
**Pull request by alice-carr**
This PR creates a concourse pipeline for deploying the lambda

relates #4

### CODE REVIEW

File: terraform/deployments/dev/main.tf [hunk 1]
```diff
@@ -6,4 +6,5 @@ module "ggroup-to-splunk-dev" {
   lambda_memory       = 256
   lambda_role_name    = "ggroup_lambda_exec_role"
   lambda_timeout      = 900
+  account_id          = "489877524855"
```
Comment (akinnane) [inline terraform/deployments/dev/main.tf]: What are they hardcoded account numbers for?

File: lambda/script.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import googleapiclient.discovery  # type: ignore
 from google.oauth2 import service_account  # type: ignore
 
-ssm_client = boto3.client("ssm")
+ssm_client = boto3.client("ssm", region_name="eu-west-2")
```
Comment (akinnane) [inline lambda/script.py]: Why is the region hardcoded?

File: terraform/deployments/dev/main.tf (hunk 1 above)
Comment (alice-carr) [reply terraform/deployments/dev/main.tf]: just removed these

Comment (akinnane) [reply terraform/deployments/dev/main.tf]: Nice @

File: lambda/script.py (hunk 2 above)
Comment (alice-carr) [reply lambda/script.py]: Removed this too

Comment (akinnane) [reply lambda/script.py]: Great

**Comment by 0atman** [comment]
I'm not seeing all the changes, surely? only 1 tf file that I can see.

Screenshot from 2020-07-29 14-40-55

**Comment by 0atman** [comment]
oh wait, this is a concourse PR, sorry! I've got tf on the brain :laughing:

File: Makefile [hunk 3]
```diff
@@ -86,3 +86,15 @@ terraform/deployments/dev/.terraform:
...
+	@cd terraform/deployments/dev; terraform init; terraform plan
+
+terraform_apply_dev:
+	@cd terraform/deployments/dev; terraform init; terraform apply -auto-approve
+
+terraform_plan_prod:
+	@cd terraform/deployments/prod; terraform init; terraform plan
+
+terraform_apply_prod:
+	@cd terraform/deployments/prod; terraform init; terraform apply -auto-approve
```
Comment (akinnane) [inline Makefile]: Missing new line at the end of the file

File: concourse/pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    icon: github-circle
+    type: git
+    source:
+      uri: https://github.com/alphagov/cyber-security-ggroup-data-to-splunk.git
+      branch: concourse
+
+  - name: health-notification
+    type: http-api
+    source:
+      uri: https://((health_host_prod))/?alert_type=concourse$&alert_name=health
```
Comment (akinnane) [inline concourse/pipeline.yml:21]: Is the `$` supposed to be in the URL?

File: Makefile (hunk 3 above)
Comment (alice-carr) [reply Makefile]: Done!

File: concourse/pipeline.yml (hunk 4 above)
Comment (alice-carr) [reply concourse/pipeline.yml:21]: Looking at other pipelines that use this health notification - yes

Comment (akinnane) [reply concourse/pipeline.yml:21]: Great!

File: Makefile (hunk 3 above)
Comment (akinnane) [reply Makefile]: Thanks

**Review by akinnane — APPROVED** [review]
LGTM

**Pull request merged**